aleo-programs/staking_lite/src/main.leo

```diff
@@ -143,14 +220,17 @@ program staking_lite.aleo {
   }
 
   transition bond_all(validator_address: address, amount: u64) {
-    assert_eq(self.caller, ADMIN);
```

Comment:
I made this permissionless. Anyone can call it. This prevents the protocol from getting stuck by the validator.